tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/AdaptiveDialogTests.cs

```diff
@@ -249,25 +249,6 @@ public async Task TestForeachWithPrompt()
             await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
         }
 
-        [Fact]
```

Comment:
We need a new test for ForEachElement.

Reply:
@mrivera-ms All the foreach tests now use ForEachElement, since ForEach has been deprecated. The change made in this pr in AdaptiveBotComponent is what makes the infrastructure use ForEachElement instead of ForEach.